lib/nfcRenderer.js

```diff
@@ -0,0 +1,752 @@
+/**
+ * VC NFC Renderer Library
+ * Handles NFC rendering for verifiable credentials.
+ * Supports both static and dynamic rendering modes.
+ *
+ * Field Requirements:
+ * - TemplateRenderMethod (W3C spec): MUST use "template" field.
+ * - NfcRenderingTemplate2024 (legacy): MUST use "payload" field.
+ */
+import * as base58 from 'base58-universal';
+import * as base64url from 'base64url-universal';
+
+const multibaseDecoders = new Map([
+  ['u', base64url],
+  ['z', base58]
+]);
+
+// ============
+// Public API
+// ============
+
+/**
+ * Check if a verifiable credential supports NFC rendering.
+ *
+ * @param {object} options - Options object.
+ * @param {object} options.credential - The verifiable credential.
+ * @returns {boolean} - 'true' if NFC is supported.
+ */
+export function supportsNFC({credential} = {}) {
+  try {
+    const renderMethod = _findNFCRenderMethod({credential});
```

Comment:
Our naming convention is usually camel case (and this will make it consistent with `renderToNfc()` below):

```suggestion
export function supportsNfc({credential} = {}) {
  try {
    const renderMethod = _findNfcRenderMethod({credential});
```